src/main/java/org/checkerframework/specimin/UnsolvedSymbolVisitor.java

```diff
@@ -1895,6 +1895,14 @@ public List<String> getArgumentTypesFromMethodCall(
         parametersList.add(((ResolvedReferenceType) type).getQualifiedName());
       } else if (type.isPrimitive()) {
         parametersList.add(type.describe());
+      } else if (type.isArray()) {
+        // the type without the [].
+        ResolvedType componentType = type.asArrayType().getComponentType();
```

Comment:
What if the component type is also an array? Can you add a test case with a 2d array to check that this will work?